Add an option to just convert a Range of a Folder

kopffarben opened the pull request:
Sometimes it makes sense in production to convert only some of the files in a folder.
I have added the option, only works if you turn off Recursive, 
otherwise it makes no sense.

bj-rn commented:
Hey, thanks. Two things:
When the file count is 1 ( min and max of the drag are both 0) it is possible to set negative values. I think this should be prevented using a Validator. And could you add a tooltip that explains what that option does?

kopffarben committed:
in Case Range.To is bigger than Range.From,
witch is possible with Keyboard (Drag don't allow)
Range.From and Range.To will be switched

kopffarben commented:
So it should fit now. 
I have added ToolTips, Range can only be edited if the count is greater than 1.

Oh yes, if From is greater than To, which you can do with the keyboard, Drag prevents this, 
From and To are swapped.

bj-rn commented:
Thanks: https://github.com/bj-rn/texconvgui/releases/tag/2.3.0

Merged your stuff idk why this is still open ... Will close it "manually"

bj-rn commented:
Ah you added another commit after I pulled ...

kopffarben commented:
last commit will only swap From/To if From > To
Without it will convert nothing in this case.

bj-rn commented:
here you go: https://github.com/bj-rn/texconvgui/releases/tag/2.3.1
